datadog_checks_dev/datadog_checks/dev/tooling/templates/integration/snmp_tile/{check_name}/README.md

```diff
@@ -0,0 +1,41 @@
+# Agent Check: snmp_{integration_name}
+
+## Overview
+
+This check monitors [{integration_name}][1].
+For details of monitored metrics see the [SNMP integration tile][2]
+
+## Setup
+
+### Installation
+
+{install_info}
+
+### Configuration
+
+1. <List of steps to setup this Integration>
+
+### Validation
+
+<Steps to validate integration is functioning as expected>
+
+## Data Collected
+
+### Metrics
+
+{integration_name} does not include any metrics.
+
+### Service Checks
+
+{integration_name} does not include any service checks.
+
+### Events
+
+{integration_name} does not include any events.
+
+## Troubleshooting
+
+Need help? Contact [Datadog support][1].
+
+[1]: https://docs.datadoghq.com/help/
+[2]: https://app.datadoghq.com/account/settings#integrations/snmp
```

Comment:
```suggestion
[2]: https://docs.datadoghq.com/network_performance_monitoring/devices/data
```

Reply:
> Docs review

Thank you @ruthnaebeck, committed your suggestions, should be good now. Can you give it another review round please?